marvel-app/features/home-page/ListCard.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  ListCard.swift
+//  marvel-app
+//
+//  Created by Michal on 02/08/2022.
+//
+
+import Foundation
+import UIKit
+import SwiftUI
+
+#if DEBUG
+struct ThisViewPreview: PreviewProvider{
+    static var previews: some View {
+        ListCard().showPreview()
+    }
+}
+#endif
+
+class ListCard: UITableViewCell{
+    static let identifier = "ListCard"
+    
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        contentView.backgroundColor = .white
+        setupLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        self.coverImageView.image = nil
+    }
+    
+    func setupLayout(){
+        self.addSubview(card)
+
+        card.anchor(top: self.topAnchor, left: self.leftAnchor, bottom: self.bottomAnchor, right: self.rightAnchor, paddingTop: 10, paddingLeft: 10, paddingBottom: 10, paddingRight: 10)
```

Comment:
Please create a constants struct and add a static variables with magic numbers to it